### ISSUE TITLE
install octave kernel

### ISSUE THREAD

**Issue by delgadom** [RhodiumGroup/docker_images#81]
so we can do stuff written in .m scripts

Mike tested this with some KE scripts and they worked!

```bash
conda install -c conda-forge octave_kernel texinfo
```

also tried this:
```bash
apt-get install --no-install-recommends  octave octave-symbolic octave-miscellaneous python-sympy  gnuplot ghostscript
```

**Comment by delgadom** [comment]
using the package manager required
```bash
apt-get install liboctave-dev
```

**Comment by delgadom** [comment]
may also need to update gnuplot?

```bash
apt-get upgrade gnuplot
```

**Comment by delgadom** [comment]
also need `ipyparallel`, then run `python -m octave_kernel install --user`

**Comment by delgadom** [comment]
actually `jupyter kernelspec` seems like a better path forward

**Comment by delgadom** [comment]
`jupyter kernelspec install /opt/conda/envs/octave/share/jupyter/kernels/octave --name octave`

**Comment by delgadom** [comment]
this apparently works best:
```apt-get install octave
conda env create -n octave texinfo
pip install octave_kernel
jupyter kernelspec install /opt/conda/envs/octave/share/jupyter/kernels/octave --name octave
```

**Comment by delgadom** [comment]
Addressed in #104
